tasks/bloom-etl/.env.template

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL=postgres://bloom-dev:bloom@localhost:5432/bloom
+EXTERNAL_ENDPOINT_URL=https://api.bloom-endpoint.org
```

Comment:
I do have a small nit with the "bloom" nomenclature, as that word primarily refers to an open source project rather than a specific implementation of that project.  It's not enough that I would ask it to be changed, but it's enough that I wouldn't want to propagate it.  I went with `EXTERNAL_ENDPOINT_URL` because it seemed descriptive in terms of what it represents and not overly restrictive (it could also be used to pull from Detroit or whatever), but I'm not necessarily stuck on that either.  

I do question whether there is value in providing a configurable path over something hardcoded given that there is no identifiable need to change that in the lifetime of this task.  I don't want to waste too many cycles solving problems that probably won't exist, especially if it would be a simple fix in another PR

Reply:
in that case, can you please add a comment on this and the same template file in backend/core saying to update EXTERNAL_blah on the backend/core side and BLOOM_blah on this one? want to make sure that developers know that there is an implicit dependency on both

also noting that this dir is literally called `bloom-etl` re. concerns around bloom naming

Reply:
Yeah, that's a fair point about the dir name.  I picked it from the beginning to match and then just never changed it.  It's been bugging me, just not enough to do anything about it.  I'll probably change it later to something more descriptive like `listings-etl` so that it's more clear what is being imported

I'll be removing the BLOOM* vars on the backend side in a subsequent PR since they won't be needed anymore, so the dependency will be short-lived enough that it shouldn't be an issue for future developers.  The shared dependency on the public site will probably be around for a while though, so I can add a note there